app/src/main/java/wordtextcounter/details/main/feature/settings/SettingsFragment.kt

```diff
@@ -3,18 +3,34 @@ package wordtextcounter.details.main.feature.settings
 import android.os.Bundle
 import com.takisoft.fix.support.v7.preference.PreferenceFragmentCompat
 import wordtextcounter.details.main.R
+import wordtextcounter.details.main.analytics.AnalyticsLogger.AnalyticsEvents.Click
+import wordtextcounter.details.main.analytics.AnalyticsLogger.AnalyticsEvents.Event
+import wordtextcounter.details.main.analytics.AnalyticsLogger.logAnalytics
 import wordtextcounter.details.main.util.Constants
 
 class SettingsFragment : PreferenceFragmentCompat() {
 
   override fun onCreatePreferencesFix(savedInstanceState: Bundle?, rootKey: String?) {
     addPreferencesFromResource(R.xml.settings)
 
+    val clipboardPref = preferenceManager.findPreference(
+        Constants.PREF_CLIPBOARD)
+    clipboardPref.setOnPreferenceChangeListener { _, _ ->
+      logAnalytics(Event("settings_clipboard_change"))
```

Comment:
How about use pref.key in place of extra strings for all prefs. Will create ext function as we add more settings

Reply:
not sure I follow.